src/lib/update-lock.ts

```diff
@@ -8,30 +7,17 @@ import * as Lock from 'rwlock';
 import * as constants from './constants';
 import {
 	ENOENT,
-	EEXIST,
 	UpdatesLockedError,
 	InternalInconsistencyError,
 } from './errors';
 import { getPathOnHost, pathExistsOnHost } from './fs-utils';
 import * as config from '../config';
+import * as lockfile from './lockfile';
 
-type asyncLockFile = typeof lockFileLib & {
-	unlockAsync(path: string): Bluebird<void>;
-	lockAsync(path: string): Bluebird<void>;
-};
-const lockFile = Bluebird.promisifyAll(lockFileLib) as asyncLockFile;
-export type LockCallback = (
-	appId: number,
-	opts: { force: boolean },
-	fn: () => PromiseLike<void>,
-) => Bluebird<void>;
+const BASE_LOCK_DIR = '/tmp/balena-supervisor/services';
```

Comment:
If you make the change to entry 
```suggestion
const BASE_LOCK_DIR = process.env.BASE_LOCK_DIR || '/tmp/balena-supervisor/services';
```